packages/web-react/package.json

```diff
@@ -4,19 +4,36 @@
   "description": "React components for web apps built with Spirit Design System",
   "license": "MIT",
   "publishConfig": {
-    "access": "public"
+    "access": "public",
+    "directory": "dist"
   },
-  "devDependencies": {
-    "@lmc-eu/browserslist-config": "^1.0.0"
+  "module": "_esNext/index.js",
+  "types": "_esNext/index.d.ts",
+  "scripts": {
+    "prebuild": "rm -rf dist && mkdir -p dist && cp package.json README.md dist/ ",
+    "build": "yarn build:umd && yarn build:es2015 && yarn build:esNext",
+    "build:umd": "yarn webpack:dev && yarn webpack:prod && yarn webpack:browser",
+    "build:es2015": "tsc --module es2015 --outDir dist/_es2015 --project ./config/tsconfig.prod.json",
+    "build:esNext": "tsc --module esNext --outDir dist/_esNext --project ./config/tsconfig.prod.json",
+    "types": "tsc --project ./config/tsconfig.json",
+    "webpack:dev": "webpack --mode development --config ./config/webpack.js --progress",
+    "webpack:prod": "webpack --mode production --config ./config/webpack.js --progress",
+    "webpack:browser": "webpack --mode production --config ./config/webpack.browser.js --progress"
   },
   "peerDependencies": {
     "@lmc-eu/spirit-web": "~0.1.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
   },
-  "dependencies": {
-    "prop-types": "^15.7.2",
+  "dependencies": {},
+  "devDependencies": {
+    "@types/react": "^17.0.19",
+    "@types/react-dom": "^17.0.9",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "ts-loader": "^9.2.5",
+    "typescript": "^4.4.2",
+    "webpack-bundle-analyzer": "^4.4.2",
```

Comment:
For documentation purposes, I think we should also list `weback` and `webpack-cli` since they are actually used in this package, no matter they are installed as a root dev dependency.

Reply:
I think that this is not possible because of that the webpack is used also in another packages.

`yarn add webpack-cli webpack --dev`

`error An unexpected error occurred: "expected workspace package to exist for \"webpack\"".`

Reply:
Yarn will refuse to add them, I mean they need to be added manually.

The aim is that developers will know what this package depends on when they open `package.json`. But I understand that manual maintenance of such thing can be too tricky.

Reply:
Well, I did it manually and it works :-)